app/lib/theme/functions/formatDates.ts

```diff
@@ -14,14 +14,16 @@ type FormatType = {
 type DTFormat = keyof FormatType;
 
 const TIMEZONE = 'America/Vancouver';
+const TIMEZONE_SHORT = 'PDT';
+
 const FORMAT_TYPE: FormatType = {
   date_year_first: 'yyyy-MM-dd',
   timestamptz: 'yyyy-MM-dd HH:mm:ss.SSSZ',
   seconds: 'MMM d, yyyy hh:mm:ss A (z)',
   minutes: 'MMM d, yyyy hh:mm A (z)',
   days_numbered: 'dd-MM-yyyy',
   days_string: 'MMMM Do, yyyy',
-  minutes_time_only: 'hh:mm A (z)'
+  minutes_time_only: 'hh:mm a (z)'
```

Comment:
We should use the abbreviated named offset `(ZZZZ)`, instead of `(z)`, which displays the IANA zone (see https://moment.github.io/luxon/#/formatting?id=table-of-tokens).
That will remove the need for the `.replace(TIMEZONE,TIMEZONE_SHORT); `, and ensure we switch seamlessly between `PST` and `PDT`